### DISCUSSION TITLE
controlling second serial Relay Board.

### DISCUSSION THREAD

**Discussion by tesau** [arendst/Tasmota#20170] [Q&A]
Howdy 
I've connected via serial a Bluetooth_Relay BLE x8  directly to a Wt32-eth01 ver 1.4 on pins TXD/IO1, RXD,IO3
It works much the same as a LC Tech Relay board .. ie serialsend5 A00300A3 .. create virtual relays then use rules 

Not using the BLE
Seems to work quite well using ethernet  

I would like to control a second relay board thru TXD/IO17, RXD/IO35  or other available IO
How would I define serial2  when sending   to second relay board
using the GUI i can define serial tx and serial rx , but  no option for numbering them
then in the console and with rules I seem to fade into the ether.

Anyone know how to use serialsend on more than one port ?

Thanks

**Comment by barbudor** [comment]
Unfortunately no more than 1 serial bridge is possible. The need hasn't  raised so much.

However as you have an ESP32, you have the marvelous Berry scripting language which allows to create more serial drivers at the expanse of a little programming. Berry is a micropython like language (similar but not exactly the same)
Please refer to the Berry page in the docs
https://tasmota.github.io/docs/Berry/

Details on the serial class here
https://tasmota.github.io/docs/Berry/#serial-class

**Reply by tesau** [reply]
Thank-You Berry Much
